ferminet/train.py

```diff
@@ -172,38 +173,39 @@ def total_energy_jvp(primals, tangents):  # pylint: disable=unused-variable
   return total_energy
 
 
-def make_training_step(mcmc_step, val_and_grad, opt_update):
```

Comment:
HMC is an instance of MCMC, so there is no need to change `mcmc` to `mc`. Please revert.

Reply:
If you want to distinguish between different MCMC algorithms, I would use `mh` for Metropolis-Hastings (used elsewhere in the code as `mh_update`)